scalameta/common/src/main/scala/org/scalameta/internal/logger.scala

```diff
@@ -0,0 +1,95 @@
+// The macro implementations are copied from https://github.com/lihaoyi/sourcecode
```

Comment:
Let's directly depend on sourcecode. That'll avoid both the scaladoc issue and the potential licensing trouble (i'm not a lawyer).